### ISSUE TITLE
Parameter CanBeVaried flag should be editable in MoBi

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#294]
After updating to 7.3, a certain parameter of the simulation is not shown in the Parameters-list of the Parameter Identification. I will send you the simulation via e-mail.

**Comment by msevestre** [comment]
This is probably due to 
https://github.com/Open-Systems-Pharmacology/MoBi/issues/170

e.g. you parameter needs to be with the flag CanBeVaried = true.

**Comment by msevestre** [comment]
and that's the commit that makes this happen
https://github.com/Open-Systems-Pharmacology/MoBi/commit/ea8c8d40482c2a61e78d3ebb99e27bb6353d787c

A parameter marked as CanBeVaried = false should not be selectable in the PI. The previous behavior was a bug.

Please confirm that this is the case

**Comment by PavelBal** [comment]
`//in MoBi, you can use any parameter except table parameters and categorial parameters`

Well it is not a table parameter and definitely not categorical...

I dont even know where I can check/change the CanBeVaried-Flag except from the PKML? This is a "normal" model created in PK-Sim/Model, the parameter was defined by me as a parameter of the simulation...

**Comment by msevestre** [comment]
you will note that the comment is gone (RED in the commit)

CanBeVaried can be changed in the parameter definition in the building block

**Comment by PavelBal** [comment]
>you will note that the comment is gone (RED in the commit)

Ahhhh yes, didnt pay attention.

>CanBeVaried can be changed in the parameter definition in the building block

You mean the "Can be varied in population"? In this case, this is very unintuitive, as I do not want to vary it in population, I just want to identify it. Furthermore, each newly created parameter is not "Can be varied in population", and excluding all of them from PI would be a strange decision, I think.

![image](https://user-images.githubusercontent.com/25903040/41537658-6f52cefe-7309-11e8-8ac4-16cf2c6d7819.png)

Additionally, the parameter "Km_GLP1" is also not "Can be varied in population", but can be selected in the PI.

**Comment by msevestre** [comment]
@PavelBal You are correct then. Can Be Varied is missing. I thought we could update that there as well but I guess not


Well that means that whomever defined that parameter in the database decided that this parameter should never be varied, even in a PI. I am not sure why this is like that to be honest :(

**Comment by msevestre** [comment]
So the question is really: WHY IS (ARE) THE PARAMETER(S) of yours CanBeVaried = false in the PKSim DB
Can you name those parameters again here ?

**Comment by PavelBal** [comment]
>Well that means that whomever defined that parameter in the database decided that this parameter should never be varied, even in a PI. I am not sure why this is like that to be honest :(

No, you misunderstand - the parameter we are talking about is user-defined (well, by me). I created it in 7.2 and everything was fine, but with 7.3, the parameter is not in the PI, while other user-defined parameters are.

**Comment by PavelBal** [comment]
It is the "k_deg_GLP" in the screenshot.

**Comment by msevestre** [comment]
oh I see. Well that's even more weird then.

**Comment by PavelBal** [comment]
OK, looking at the "Group" of the parameter, I now remember:

This was a metabolization reaction from PK-Sim, and this was the "kcat_app" (or how it is called) which I later renamed (because I do not need all those kcat_app/km_app) in "k_deg_GLP". So while the behavior is correct - the "apparent" kcat should not be variated because it is a function of kcat (which will be identified) and an "Inhibition factor - it is actually misleading because a) there is no way to change the "CanBeVaried" in MoBi and b) this change wasnt really documented :)

**Comment by msevestre** [comment]
ok renaming
